test/mlir/krnl/krnl_bundle_lstm_memory_pool.mlir

```diff
@@ -30,8 +30,8 @@ module {
     %7 = "krnl.getref"(%6, %c0_i64, %3) : (memref<?xi8>, i64, index) -> memref<1x?x3xf32>
     %8:3 = krnl.define_loops 3
     krnl.iterate(%8#0, %8#1, %8#2) with (%8#0 -> %arg3 = 0 to 1, %8#1 -> %arg4 = 0 to %1, %8#2 -> %arg5 = 0 to 3) {
-      affine.store %cst_1, %2[%arg3, %arg4, %arg5] : memref<1x?x3xf32>
-      affine.store %cst_1, %7[%arg3, %arg4, %arg5] : memref<1x?x3xf32>
+      krnl.store %cst_1, %2[%arg3, %arg4, %arg5] : memref<1x?x3xf32>
```

Comment:
Could you take this test, and feed it to krnl to affine, and make sure all of the krnl stores are transformed to the affine as seen on the original test? Visual inspection would be enough... if you wanted to make it an official test, that would be fine too.

Reply:
@AlexandreEichenberger I fed it krnl-to-affine and saw all krnl.load/store were transformed to affine.load/store as seen on the original test. Memory pool bundling is done at the krnl level, so it is good to see that krnl.load/store does not affect the correctness of the bundling here, instead of lower to affine.

Reply:
great, thanks for visually inspecting the code.